assets/queries/openAPI/parameter_object_schema_content/query.rego

```diff
@@ -0,0 +1,45 @@
+package Cx
+
+import data.generic.openapi as openAPILib
+
+CxPolicy[result] {
+	doc := input.document[i]
+	openAPILib.checkOpenAPI(doc) != "undefined"
+	params := doc.paths[name].parameters[n]
+	object.get(params, "$ref", "undefined") == "undefined"
```

Comment:
Parameters can be defined in other attributes - https://swagger.io/docs/specification/describing-parameters/#path-parameters. For example, path[name].get.parameters